src/wallet/rpcwallet.cpp

```diff
@@ -2269,7 +2269,7 @@ UniValue walletpassphrase(const JSONRPCRequest& request)
             "This is needed prior to performing transactions related to private keys such as sending bitcoins\n"
             "\nArguments:\n"
             "1. \"passphrase\"     (string, required) The wallet passphrase\n"
-            "2. timeout            (numeric, required) The time to keep the decryption key in seconds.\n"
+            "2. timeout            (numeric, required) The time to keep the decryption key in seconds. Limited to at most 2147483647 (2^31 - 1) seconds.\n"
```

Comment:
The claming semantics need to be in the documentation.

Reply:
Done.